### PR TITLE
Update dependencies to latest version

### PR DESCRIPTION
**Pull request by Kemosabert**
### Description
Update all the project's dependencies to the latest version. 

### Breaking changes
none

### CODE REVIEW

**Comment by duivvv** [comment]
Why are we all updating them explicitly?
https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004

We only need to update the dependencies if the major changed or < 1.0.0

File: package.json [hunk 1]
```diff
@@ -63,79 +63,79 @@
...
-    "postcss-loader": "^2.0.7",
-    "postcss-mixins": "^6.1.0",
+    "postcss-import": "^11.1.0",
+    "postcss-loader": "^2.1.5",
+    "postcss-mixins": "^6.2.0",
     "postcss-nested": "^3.0.0",
     "postcss-pseudoelements": "^5.0.0",
     "postcss-reporter": "^5.0.0",
-    "prettier": "^1.4.4",
+    "prettier": "^1.13.5",
```
Comment (nickwaelkens) [inline package.json]: This will result in a follow-up "fix linting" PR 💃

**Comment by nickwaelkens** [comment]
Agreed with @duivvv - feels kinda useless, semver tackles this for us.

**Comment by InstaK** [comment]
I guess this PR can be reduced to only the hardcoding of prettier? So without the caret then?

**Comment by duivvv** [comment]
@InstaK 

+ having a look at what dependencies moved to a major release (and testing them)

**Comment by Kemosabert** [comment]
Addressed this issue in a new PR: #292

**Pull request closed without merging**